internal/grid/types.go

```diff
@@ -39,6 +47,14 @@ func (m *MSS) Get(key string) string {
 	return (*m)[key]
 }
 
+// Set a key, value pair.
+func (m *MSS) Set(key, value string) {
+	if m == nil {
+		*m = mssPool.Get().(map[string]string)
```

Comment:
We are getting from the pool here, shouldn't we be clearing the contents before using it for the first time?

Reply:
Or better yet, call `NewMMS()` ?

Reply:
I will clear it. Avoids an alloc.

Reply:
Going to merge this, please send a new PR @klauspost